framework/src/org/checkerframework/framework/util/defaults/QualifierDefaults.java

```diff
@@ -119,6 +119,22 @@
     };
 
     /**
+     * Standard unchecked default locations that should be top
+     */
+    // Fields are default to top so that warnings are issued at field reads, which we believe are more common
+    // than field writes. Future work is to specify different defaults for field reads and field writes.
```

Comment:
This would still be the default if writes were more common than reads, right? My understanding is that fields are defaulted to top because no assumptions can be made about their type qualifier hence the least specific possible qualifier is used, whereas parameters are defaulted to bottom because no assumptions can be made about the contract of the method regarding a given parameter, hence the most specific qualifier (and therefore the most specific contract) is used.